docs/products/compute/compute-instances/guides/placement-groups/index.md

```diff
@@ -0,0 +1,213 @@
+
+## Overview
+
+The Placement Groups service gives you a convenient way to set up groups of your compute instances, using Cloud Manager, API operations, or our CLI. Create a new placement group in a supported region and add new or existing compute instances from that region to your group. With the new group created, we physically move your compute instances into it, based on your desired model.
+
+## Availability
+
+<!--
+
+**CV: 03/2024** Availability at limited availability is only on two data centers - Washington D.C., and Miami. We'll be progressively opening up more regions as they're qualified. The request is to hide availability information in the docs, prior to actual GA.
+
+- UI: There's a mouse-over tooltip that reveals the available regions when creating a placement group. It'll be dynamically updated as new regions are made available.
+- API: The `GET /v4/regions:id` operation in the API spec lists `Placement Group` in the `capabilities` array, if PGs are supported for that region. This is the only op I see that shows support/non-support. The issue here is, you need to know the `id` for the target region, in order to run this operation. How would this work in an automated API workflow? Would you set up automation to periodically ping all regions, until it sees that `Placement Groups` is listed for a specific region? (In which case, the workflow could be manipulated to include the new region.)
+
+Because of the issue with the API, I'm including the Availability section in the docs. It's easy enough to update the docs on a weekly release cadence as we add new supported regions. (Do we anticipate qualifying regions quicker than once a week?)
+
+-->
+
+The Placement Groups service is available in select regions. Currently, this includes:
+
+- Miami, FL (us-mia)
+
+- Washington, DC (us-iad)
+
+{{< note >}}
+Placement Groups is in limited availability. Throughout this phase, we expect to increase the number of supported regions.
+{{< /note >}}
+
+## Affinity, enforcement, and compliance
+
+To distribute your compute instances in a placement group to meet your model, we use the industry-recognized affinity standard. You can pick from one of two types:
+
+- **Affinity**. Your compute instances are physically close together, possibly on the same host. Set this as your preferred container to help with performance.
+
+- **Anti-affinity**. Your compute instances are placed in separate fault domains, but they're still in the same region. Use this preferred container to better support a high-availability model.
+
+You also select how your affinity type should be enforced, when adding more compute instances to it in the future. This determines whether or not the placement group remains accessible – is “compliant”-- after you add a new compute instance:
```

Comment:
Suggestion:

> In addition to selecting the affinity type, you also select the how the affinity type is enforced when additional Compute Instances are added to the placement group.

It's also not clear what _accessible_ or _compliance_ means here. To me, accessible means that the instances are _network-accessible_, but I don't think that is what's meant here.